### DISCUSSION TITLE
Good news / bad news: Higher output power coming, but the schematic will slightly change

### DISCUSSION THREAD

**Discussion by EngineerGuy314** [EngineerGuy314/pico-WSPRer#38] [Announcements]
tldr: if you are using the 3 GPIO in parallel to low side drive the GPS Module you will have to shift from IO 8,9,10 to 10,11,12 for the latest program.

I have successfully tested doubling the output power by using 4 outputs instead of 2. Basically GP6 and GP8 will be in parallel driving one leg of the dipole, and GP7 and GP9 will drive the other.

Because of limitations in the PIO state-machine the RF outputs I use must be adjacent to each other, so I can't just pick any unused IO. The low side drive outputs will have to be moved. If your GPS is always on (base station) or you are using the MOSFET method than you can ignore this change. And if you do not need the additional RF power, existing designs that use GP6 and GP7 for RF will still work as they are. The custom PCB design will also be changed to make use of the extra RF output.

In my testing, the current setup of GP6 and GP7 into a 70 ohm resistor put out 27mW of total power (that is *total* power, including all harmonics and phase noise). With the 4 outputs I measured 59mW, so this will hopefully lead to some more spots.  The "real" power output of the signal in the 20M band should increase from about 10mW to 20mW.

**Comment by whallmann** [comment]
Nice work Rob!
